src/frameName.cpp

```diff
@@ -167,6 +167,7 @@ void FrameName::javaMethodName(jmethodID method) {
     char* method_sig = NULL;
 
     jvmtiEnv* jvmti = VM::jvmti();
+    JNIEnv* jni = VM::jni();
```

Comment:
Similar to above: obtain `JNIEnv*` once in `FrameName` constructor.

Reply:
Cached as above.